test/test_car_parser.py

```diff
@@ -184,5 +185,12 @@ def test_car_parser(self):
                 else:
                     self.assertEqual(cont[key], cases[car][key], '%s != %s for key %s' % (cont[key], cases[car][key], key))
 
+    def test_empty_xml(self):
+        """
+        Check that correct exception is raised for an XML file with no records.
+        """
+        parser = CarParser("<something></something>")
```

Comment:
I feel like this doesn't just test for a XML file with no records, but just checks that garbage XML raises the exception.

Would the message below also raise the exception?
```
<?xml version="1.0" encoding="UTF-8"?>
<urf:UsageRecord xmlns:urf="http://eu-emi.eu/namespaces/2012/11/computerecord" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" xsi:schemaLocation="http://eu-emi.eu/namespaces/2012/11/computerecord ../../../org.glite.dgas_B_4_0_0/config-consumers/car_v1.2.xsd ">
</urf:UsageRecord>
```

Reply:
I'll clarify the wording in that test.

That message doesn't raise an exception. Not sure what it should do, but I don't want to dive down the car_parser rabbit hole right now. I'll add a commented out test for it though.

Reply:
looks good